apps/pie-microsite/src/_11ty/filters/pieIconsSvg.js

```diff
@@ -1,9 +1,25 @@
-const pieicons = require('@justeattakeaway/pie-icons');
+const pieIcons = require('@justeattakeaway/pie-icons');
+
+
+// eslint-disable-next-line consistent-return
+const getIconByName = (iconName, iconAttributes) => {
+    try {
+        return pieIcons.default.icons[iconName].toSvg(iconAttributes);
+    } catch (error) {
```

Comment:
I know I added the original catch - should this just break the build? Honest question, not a suggestion!

Reply:
will do as a separate pr